src/main/java/edu/harvard/iq/dataverse/DataFile.java

```diff
@@ -1111,4 +1112,8 @@ private boolean tagExists(String tagLabel) {
         }
         return false;
     }
+    @Override
+    public PidProvider getEffectivePidGenerator() {
+        return getOwner().getEffectivePidGenerator();
```

Comment:
I hope this transitive implementation of the property is in line with planned developments for more advanced filePIDs.